service/history/workflow/update/update.go

```diff
@@ -54,6 +55,21 @@ const (
 	stateCompleted
 )
 
+func stateToString(s state) string {
```

Comment:
If you implement `Stringer` instead then you don't need to explicitly call it in `fmt.Sprintf`.